src/components/training/UpcomingTrainingSection.js

```diff
@@ -1,14 +1,31 @@
 import React from 'react'
 import Section from '../layout/Section'
 import Grid, { Col, Row } from '../layout/Grid'
-import { H2Ref, H3, P } from '../text'
+import { H2Ref, H3, P, H5 } from '../text'
 import { TrainingItem, TrainingList } from './'
 import moment from 'moment'
 import Link from '../navigation/Link'
-
+import styled from 'styled-components'
 import { selectTrainings } from '../../config/data'
 import Newsletter from '../elements/Newsletter'
 
+import CorporateTrainingCard from '../elements/CorporateTrainingCard'
+
+const CorporateCrossSell = styled.div`
+  border: 3px dotted #dedede;
```

Comment:
@shakersinthedark I think we should avoid hardcoding colors. Is this a color that we already use somewhere else? if it's a new color, can you move the color to a variable here https://github.com/reactjsacademy/reactjsacademy/blob/0c07a5044acdf69766e8a7f7e26bc9b0af50efcf/src/config/styles.js? However, my understanding is that at this point in the development of this site the design system we've built should probably include any colors we need